frontend/src/components/NewGroup.tsx

```diff
@@ -210,7 +210,7 @@ function NewGroup(props: { onClose: () => void }) {
                 setSelectedFriends([]);
 
                 // TODO(tommy): Send an API Request to the server to create a new group
-
+                sendPostRequestToServer(newGroup);
```

Comment:
Could you make it so that it closes the modal after it sends a successful POST request?